Selenium.Axe/Selenium.Axe.Test/AxeBuilderTest.cs

```diff
@@ -31,36 +31,40 @@ public class AxeBuilderTest
             url = "www.test.com"
         };
 
-        [TestInitialize]
+        [OneTimeSetUp]
         public void TestInitialize()
         {
             webDriverMock = new Mock<IWebDriver>();
             jsExecutorMock = webDriverMock.As<IJavaScriptExecutor>();
             targetLocatorMock = new Mock<ITargetLocator>();
         }
 
-        [TestMethod]
-        [ExpectedException(typeof(ArgumentNullException))]
+        [Test]
         public void ThrowWhenDriverIsNull()
         {
-            //arrange / act /assert
-            var axeBuilder = new AxeBuilder(null);
-            axeBuilder.Should().NotBeNull();
+            Assert.Throws<ArgumentNullException>(() =>
+            {
+                //arrange / act /assert
+                var axeBuilder = new AxeBuilder(null);
```

Comment:
nit: this and `ThrowWhenOptionsAreNull` can be simplified a bit; the `Should().NotBeNull()` assertion isn't doing anything here, it's just the constructor invocation that needs to be within the lambda body